app/webpack/users/edit/components/applications.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import PropTypes from "prop-types";
+import moment from "moment";
+
+import SettingsItem from "./settings_item";
+
+const Applications = ( { showModal, apps = [] } ) => {
+  const renderEmptyState = ( ) => (
+    <p className="col-xs-9">
+      {I18n.t( "you_have_not_authorized_any_applications", { site_name: SITE.name } )}
```

Comment:
You need to add this to `en.yml`, and I don't think it needs a `site_name` variable.